src/Model/User.php

```diff
@@ -466,19 +466,23 @@ public static function create(array $data)
 		// collapse multiple spaces in name
 		$username = preg_replace('/ +/', ' ', $username);
 
-		if (mb_strlen($username) > 48) {
-			throw new Exception(L10n::t('Please use a shorter name.'));
+		$username_min_length = max(1, min(255, intval(Config::get('system', 'username_min_length', 0))));
+		$username_max_length = max(1, min(255, intval(Config::get('system', 'username_max_length', 0))));
+
+		if (mb_strlen($username) < $username_min_length) {
+			throw new Exception(L10n::tt('Username should be at least %s character.', 'Username should be at least %s character.', $username_min_length));
```

Comment:
The plural should have the plural "s", shouldn't it?

Reply:
Oops.